source/assets/stylesheets/styles/_project_story_large_tile.scss

```diff
@@ -0,0 +1,95 @@
+.project-story-large-tile {
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 6px;
+  padding-right: 6px;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    margin-bottom: 0;
+    width: 66%;
+  }
+}
+
+.project-story-large-tile__container {
+  background-color: $project-story-tile__background-color;
+  box-sizing: border-box;
+  border: 1px solid $project-story-tile__border-color;
+  padding: 6px;
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
+
+.project-story-large-tile__image {
+  display: block;
+  height: 125px;
+  object-fit: cover;
+  width: 100%;
+
+  @include breakpoint($max-mobile-container) {
+    height: 180px;
+  }
```

Comment:
As mentioned in a [previous comment](https://github.com/unboxed/ubxd_web_refresh/pull/49#issuecomment-158009519), I think we should allow the images to maintain their ratios, so for the mobile view we would only need a 100% width block element, but from tablet upwards we should add the `object-fit` and a specified height to contain it.